content/v2/tlds.markdown

```diff
@@ -47,6 +47,16 @@ List all TLDs.
 <%= pretty_print_fixture("/listTlds/success.http") %>
 ~~~
 
+### Sorting
+
+For general information about sorting, please refer to the [main guide](/v2/#sorting).
+
+Name | Description
+-----|------------
+`tld` | Sort TLDs by tld
```

Comment:
Actually, as this is the only possible value, and there is no alternative, we may simply want to omit it. Same for webhooks.